build.gradle

```diff
@@ -5,13 +5,14 @@ buildscript {
   repositories {
     google()
     jcenter()
+    maven { url 'https://jitpack.io' }
   }
   dependencies {
     classpath 'com.android.tools.build:gradle:3.2.1'
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
     classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
-    classpath 'com.facebook.testing.screenshot:plugin:0.8.0'
+    classpath 'com.github.tiagoengel:screenshot-tests-for-android:fix-reference-dir-SNAPSHOT'
```

Comment:
Branch with the fix to just compare screenshots and not record them (we do that on test lab now). I'll try to get this merged back into the main branch in due time.